Baidu Scholar.js

```diff
@@ -106,16 +110,63 @@ function scrape(doc, url) {
 				item.abstractNote = text(doc, 'div.sc_abstract') || text(doc, 'p.abstract');
 			}
 			item.attachments.push({
```

Comment:
what's wrong with the original here? We'd definitely want "Snapshot" to be part of the title to ensure uniform behavior across Zotero.

Reply:
OK